engine/api/src/main/scala/pl/touk/nussknacker/engine/api/dict/static/StaticDictDefinition.scala

```diff
@@ -0,0 +1,78 @@
+package pl.touk.nussknacker.engine.api.dict.static
+
+import pl.touk.nussknacker.engine.api.dict.{DictDefinition, ReturningKeyWithoutTransformation}
+import pl.touk.nussknacker.engine.api.typed.ClazzRef
+import pl.touk.nussknacker.engine.api.typed.typing.{SingleTypingResult, TypedClass}
+
+import scala.reflect.ClassTag
+
+/**
+ * It is DictDefinition which contains embedded label <> key transformation.
+ */
+trait StaticDictDefinition extends DictDefinition {
+
+  def labelByKey: Map[String, String]
+
+  lazy val keyByLabel: Map[String, String] = labelByKey.map(_.swap)
+
```

Comment:
Do we check somewhere if the map is invertible?

Reply:
No, we don't. I can put some assertion for example that `labels` are unique. WDYT?

Reply:
+1

Reply:
added